tasks/install.yml

```diff
@@ -2,24 +2,26 @@
 
 - name: MySQL | Installing dependencies
   apt:
-    pkg: "{{ item }}"
+    pkg: "{{ mysql_required_libs }}"
     state: present
-  with_items: "{{ mysql_required_libs }}"
 
 - name: MySQL | Download MySQL config deb
   get_url:
     url: "{{ mysql_deb_repo }}"
-    dest: /tmp
+    dest: /usr/src
     validate_certs: "no"
+  when: mysql_flavour == 'mysql'
 
 - name: MySQL | Install MySQL repo deb
   apt:
-    deb: "/tmp/{{ mysql_deb_package }}"
+    deb: "/usr/src/{{ mysql_deb_package }}"
+  when: mysql_flavour == 'mysql'
```

Comment:
I prefer diferent task files over zillions of `when` statements

Reply:
Ok, changed.